### DISCUSSION TITLE
Is it possible to load a json file from a link url?

### DISCUSSION THREAD

**Discussion by RitaAguiarAdsk** [cityjson/specs#114] [Q&A] [answered]
Is it possible to load a json file from a link url with `cityjson.load(path)`?
For example, I want to load this file, but I don't want to download it, I want to read it directly from the web:
https://s3.dualstack.us-east-1.amazonaws.com/opencitymodel/2019-jun/json/NewYork/36061/NewYork-36061-000.json

**Comment by balazsdukai** [comment, marked as answer]
No, that is currently not possible with cjio (I think that what you are referring to, right?). You could submit a feature request at https://github.com/cityjson/cjio/issues

**Reply by RitaAguiarAdsk** [reply]
Thank you for your reply. Yes, I wanted to access the data in a file stored online using cjio.
